corehq/apps/locations/static/locations/js/location_drilldown.async.js

```diff
@@ -79,7 +79,7 @@ function LocationSelectViewModel(hierarchy, default_caption, auto_drill, loc_fil
 
     // load location hierarchy and set initial path
     this.load = function(locs, selected) {
-        this.root(new model.func({name: '_root', children: locs}, this));
+        this.root(new model.func({name: '_root', children: locs}, this, undefined, undefined, undefined, loc_url));
```

Comment:
I didn't like this either. This is used in lot of other places (some of which I don't have context). I ended up refactoring more than I had time for, by this point, so I didn't go on fixing this. I will spend an hour more and see if it can be down without breaking other things.